<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlay</class>
 <widget class="QWidget" name="GamePlay">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QPushButton" name="pbStart">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>49</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Â¡Comenzar!</string>
   </property>
  </widget>
  <widget class="QWidget" name="questionsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>361</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="question_layout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="questionTitle_layout">
     <item>
      <widget class="QLabel" name="labeltitle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="optionsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>661</width>
     <height>981</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>971</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="options_layout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="lblResultado">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="scoreWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>829</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="scoresLayout">
     <item>
      <widget class="QLabel" name="labelScore">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SCORE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="teScore">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>200</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbItemEnd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Siguiente</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>questionsWidget</zorder>
  <zorder>optionsWidget</zorder>
  <zorder>pbStart</zorder>
  <zorder>scoreWidget</zorder>
  <zorder>lblResultado</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
